src/pages/gounSeo/Main/Main.js

```diff
@@ -1,9 +1,167 @@
 import React from 'react';
 import './Main.scss';
-
+import Comment from './Comment/Comment';
 class MainGoUn extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      reply: '',
+      commentArr: [
+        { id: 'canon_mj', text: '베이킹 클래스 즐거웠어~' },
+        { id: 'cocoadidi', text: '다음에 또 같이 하자' },
+      ],
+    };
+  }
+
+  handlereplyInput = event => {
```

Comment:
```suggestion
  handleReplyInput = event => {
```

사소한 부분이지만, camelCase로 잘 작성해주세요!